options.go

```diff
@@ -163,27 +144,38 @@ func loadEnv(opts *Options) {
 	opts.Env.Contents = []byte(value)
 }
 
-func findManifest(filename string) ([]byte, string, bool, error) {
+func (opts *Options) findManifest() error {
```

Comment:
I feel that it's worth giving the name that represents that state is changes, like `initializeManifest` or `configureManifest`